packages/api/project.json

```diff
@@ -50,6 +50,36 @@
       "options": {
         "lintFilePatterns": ["packages/api/**/*.{ts,tsx,js,jsx}"]
       }
+    },
+    "prisma-generate": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "npx prisma generate --schema=packages/api/prisma/schema.prisma"
+      }
+    },
```

Comment:
Two major things have to be adjusted from the standard set up so that the prisma schema and client can be within the package rather than in the root of the repo. First we need to define the common prisma commands in the project file. This solves a few problems.

1. Prisma expects the .env file to be stored in the root of the repo. If nx executes prisma, then it can load the environment for us from the .env file in the api package
2. Prisma expects the schema to be at prisma/schema.prisma. These commands allow us to set that path so we don't have to type that in every time we want to execute a prisma command.

The downside is that for every prisma command you want to use, you have to add it here rather than having one that you can pass multiple arguments to. This is because nx and prisma handle argument parsing differently.